packages/js-moi-logic/src.ts/logic-base.ts

```diff
@@ -185,6 +173,16 @@ export abstract class LogicBase extends ElementDescriptor {
     protected createIxRequest(ixObject: LogicIxObject): LogicIxRequest {
         const unwrap = async () => {
             const ix = await ixObject.call();
+            const error =
+              "error" in ix.receipt.extra_data &&
+              ix.receipt.extra_data.error != "0x"
+                ? ManifestCoder.decodeException(ix.receipt.extra_data.error)
```

Comment:
decode exception itself check for` ix.receipt.extra_data.error != "0x"`.

Reply:
https://github.com/sarvalabs/js-moi-sdk/pull/75/commits/13b0708ef769be3ddb23de85975b805f491f7163